src/client/components/SearchFilterComponent/SearchFilter.styles.css

```diff
@@ -0,0 +1,210 @@
+.search-filter-container {
+  display: flex;
+  flex-direction: column;
+  width: 387px;
+  padding: 1rem;
+  height: 1002px;
+  border-radius: 1.2rem;
+}
+.search-filter-container .title {
+  width: 320px;
+  height: 53px;
+  margin-top: 1.5rem;
+}
+.search-filter-container h1 {
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+  font-size: 20px;
+  line-height: 21px;
+  width: 340px;
+  height: 40px;
+  padding-top: 0.5rem;
+  padding-left: 2.5rem;
+  position: relative;
+  right: 2.1rem;
+  border-style: solid;
+  border-width: 0.1rem;
+  border-color: #c4c4c4;
+}
+
+.search-filter-container h1 #a {
+  position: relative;
+  left: 14rem;
+  width: 10px;
+}
+.search-filter-container h1 #b {
+  position: relative;
+  left: 10rem;
+  width: 10px;
+}
+.search-filter-container h1 #c {
+  position: relative;
+  left: 5.5rem;
+  width: 10px;
+}
+
+hr {
+  position: relative;
+  margin-top: 2rem;
+  right: 0.9rem;
+  width: 22rem;
```

Comment:
> in the figma design this line is not 100% i mean the size of the line is not the same width of the component you can look at it
![foto linea](https://user-images.githubusercontent.com/66638147/108269147-dbec4300-716d-11eb-971f-586331056f45.PNG)

Reply:
yes, but then the outer container can have a padding, which is giving the space between it's content and the border of the container. So what this will do is that the line will take the width from inside the padding of the outer container.